descriptions/rep/pt_br.lua

```diff
@@ -1,6 +1,7 @@
 -------------------------------------------------------------
 -----  Basic Portuguese (Brazil) descriptions based on  -----
 -----                 Platinumgod.co.uk 				-----
+-----             By: LuanRoger and NotZin02            -----
```

Comment:
@wofsauge NotZin02 helped me with the translation, please add credits to it too :)